zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/ObjectenApiPlugin.kt

```diff
@@ -61,4 +61,8 @@ class ObjectenApiPlugin(
     fun objectUpdate(objectUrl: URI, objectRequest: ObjectRequest): ObjectWrapper {
         return objectenApiClient.objectUpdate(authenticationPluginConfiguration, objectUrl, objectRequest)
     }
+
+    fun objectCreate(objectUrl: URI, objectRequest: ObjectRequest): ObjectWrapper {
```

Comment:
I wonder why we started doing objectUpdate instead of updateObject, as the standard is the other way around.

Please change this to createObject, and I'll check on what to do with objectUpdate